internal/scorecard/alpha/run_test.go

```diff
@@ -100,3 +101,92 @@ func TestRunTests(t *testing.T) {
 
 	}
 }
+
+// TODO(joelanford): rewrite to use ginkgo/gomega
+func TestRunParallelPass(t *testing.T) {
+	scorecard := getFakeScorecard(true)
+	ctx, cancel := context.WithTimeout(context.Background(), 7*time.Millisecond)
```

Comment:
Any particular reason for this timeout? This could flake on a slow machine.

Reply:
We can make the durations longer if we see flakes. But the idea is to run two 5ms "tests" (i.e. sleeps) concurrently and see that they finish before 7ms has passed. This test basically ensures that setting parallel _actually_ runs the tests in parallel rather than sequentially.